rust/rebase_witness_sdk/src/client.rs

```diff
@@ -62,15 +67,15 @@ impl Client {
     pub async fn statement(&self, req: StatementReq) -> Result<FlowResponse, ClientError> {
         let client = HttpClient::new();
 
-        Ok(client
+        client
```

Comment:
don't need an `Ok` here?

Reply:
Clippy was warning me it was redundant since I end in a `?`.

Reply:
in clippy we trust